src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -112,7 +112,8 @@ public function render(array $data, ContextInterface $context)
             'disabled' => null,
             'val' => null,
             'idPrefix' => null,
-            'templateVars' => []
+            'templateVars' => [],
+            'label' => true
```

Comment:
Because I deemed it not important within the widget as within that scope it's label